deployments/helm-chart/templates/controller-deployment.yaml

```diff
@@ -80,12 +80,12 @@ spec:
           - -report-ingress-status
```

Comment:
` - -external-service={{ .Values.controller.reportIngressStatus.externalService }}` will result in a bug.
This has to be pointing to the new Service name, otherwise no ingresses would have the `ADDRESS`.
`- -external-service={{ include "nginx-ingress.name" . }}`

Reply:
Excellent catch. 

I changed the generation of the service name as we want that independently configurable to the name of the nginx-ingress. 

Thanks

Reply:
This will not allow customization of the service name via `controller.reportIngressStatus.externalService`. Is the plan to deprecate that?

Reply:
@Dean-Coakley know only cause I banged my head against this for couple hours :D (had to copy to chart locally to do same namespace change)

Reply:
@paigr Yeah, I'll change it back to what I had 😆 
https://github.com/nginxinc/kubernetes-ingress/pull/546/commits/a4d7ffd58b229f657eb8158d1616d071cf908c17